src/ui/controls/shortcut/index.tsx

```diff
@@ -18,6 +18,18 @@ interface ShortcutLinkProps {
 export default function ShortcutLink(props: ShortcutLinkProps) {
     const cls = mergeClass('shortcut', props.class);
     const shortcut = props.shortcuts[props.commandName];
+    const shortcutClass = document.getElementsByClassName("shortcut");
+    const toggleCurrent = document.getElementsByClassName("shortcut").item(0);
+    const toggleExtension = document.getElementsByClassName("shortcut").item(1);
+
+    if (shortcutClass.length == 3) {
+        if (toggleCurrent.innerHTML.lastIndexOf("Toggle") > 5) {
+            toggleCurrent.innerHTML = toggleCurrent.innerHTML.slice(0,toggleCurrent.innerHTML.indexOf("Toggle",1));
```

Comment:
Use of `innerHTML` is discouraged.

Reply:
My logic for this issue consists of the following
1. If node.content is equal to "..." then set it to the initial text
2. If node.content is equal to "Toggle" then do not set the node context node as this means that it was already created by the re render function.

But the issue that I ran into was that there was no actual way to get the node.content so I grabbed the InnerHTML.